### DISCUSSION TITLE
Delete cash account without deleting transactions

### DISCUSSION THREAD

**Discussion by pwsoftwaredesigns** [firefly-iii/firefly-iii#5676] [Firefly III]
Originally, I had created a "cash" account to track my cash.  I would transfer money out of my checking account into my cash account.  Since I don't actually us cash that often, and I read on the documentation that there is a built-in (cash) account for this type of thing, I was just going to delete the account.  However, I did a test to see what would happen in another instance, and it looks like any transfers I made in the pas from the checking account will appear back in the checking account.

I also noticed that there is the option to move all transactions, on the account being deleted, to another account.  My question is, is there a way to move these transactions to the internal (cash) account?

**Comment by JC5** [comment]
This isn't possible yet, I'll write it down for the new layout :+1:
